### PR TITLE
Increase cpu requests limit

### PR DESCRIPTION
**Pull request by vikasmb**
- Porting changes from release-1.10 branch made in PR #1749

**What type of PR is this?**

cleanup
**Which issue does this PR fix**:
Porting CPU requests change made in 1.10 branch in #1749

**What does this PR do / Why do we need it**:
Increases CPU requests to 25m

**If an issue # is not available please add repro steps and logs from IPAMD/CNI showing the issue**:

**Testing done on this change**:

**Automation added to e2e**:

**Will this PR introduce any new dependencies?**:

No
**Will this break upgrades or downgrades. Has updating a running cluster been tested?**:
No

**Does this change require updates to the CNI daemonset config files to work?**:

Yes
**Does this PR introduce any user-facing change?**:

No
```release-note

```

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: charts/aws-vpc-cni/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: aws-vpc-cni
-version: 1.1.17
+version: 1.1.18
 appVersion: "v1.11.2"
```
Comment (achevuru) [inline charts/aws-vpc-cni/Chart.yaml:4]: We need to bump the appVersion to 1.11.3 as well, so maybe we can update that and merge this PR as part of release process.

**Review by achevuru — APPROVED** [review]

File: charts/aws-vpc-cni/Chart.yaml (hunk 1 above)
Comment (vikasmb) [reply charts/aws-vpc-cni/Chart.yaml:4]: Synced up offline, appVersion change would be made in a separate PR.

**Comment by jayanthvn** [comment]
Even for the cni-metrics helper we need to bump up the version.

**Comment by vikasmb** [comment]
> Even for the cni-metrics helper we need to bump up the version - https://github.com/vikasmb/amazon-vpc-cni-k8s/blob/master/charts/cni-metrics-helper/Chart.yaml

@jayanthvn In previous change for this file - https://github.com/aws/amazon-vpc-cni-k8s/pull/1962/files, it would be changed along with image version, is that accurate ? If so, I will do that change while changing the app version in a separate PR.

**Pull request merged**